students/jihyun/westagram/users/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+
+# Create your models here.
+class Name(models.Model) :
+    name = models.CharField(max_length=50)
+
+    class Meta :
+        db_table = 'names'
+
+class Email(models.Model) :
+    address = models.CharField(max_length=100)
+    name = models.ForeignKey(Name, on_delete=models.CASCADE)
+
+    class Meta :
+        db_table = 'emails'
+
+class Password(models.Model) :
+    password = models.CharField(max_length=20)
+    email = models.ForeignKey(Email, on_delete=models.CASCADE)
+
+    class Meta :
+        db_table = 'passwords'
+
+class Phonenumber(models.Model) :
+    phonenumber = models.CharField(max_length=20)
+    name = models.ForeignKey(Name, on_delete=models.CASCADE)
+
+    class Meta :
+        db_table = 'phonenumbers'
```

Comment:
이름, 이메일, 비밀번호를 전부 테이블로 분리한 이유가 있으신가요?
사용자 정보를 모두 포함하고 있는 사용자(User) 테이블로 합쳐주세요~ 

```suggestion

class User(models.Model) :
    name = models.CharField(max_length=50)
    address = models.CharField(max_length=100)
    password = models.CharField(max_length=20)
    email = models.ForeignKey(Email, on_delete=models.CASCADE)
    phonenumber = models.CharField(max_length=20)
            
    class Meta :
        db_table = 'users'
```